extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -702,11 +693,16 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
       const offsetTop = input.outerHeight()! + offset.top; // both are in the template
       const bottomGap = 10;
       this.view.S.cached('contacts').css({
-        display: 'none',
+        display: renderableContacts.length > 0 ? 'block' : 'none', // show popup only when there are renderableContacts
         left: leftOffset,
         top: offsetTop,
         maxHeight: `calc(100% - ${offsetTop + bottomGap}px)`,
       });
+    } else {
+      this.view.S.cached('contacts').find('ul').html('<li>No Contacts Found</li>'); // xss-direct
+      if (!this.googleContactsSearchEnabled) {
+        this.addBtnToAllowSearchContactsFromGoogle(input);
```

Comment:
I changed this logic because with previous logic, when there's search result for something like `e2e` and then user continues type like `e2eaaaaaaaa`, it still displays `e2e` result.

Reply:
I have tested it and it's looking ok. So, if `googleContactsSearchEnabled` is `true` and I'm typing a non-existing name `aaaaaa`, nothing is displayed (`No Contacts Found` isn't displayed).
However if I first type an existing name like `flow`, wait until the contacts appear and then continue typing to `flowwww`, then `No Contacts Found` is displayed replacing the found contacts. Is this the desired behaviour?

Reply:
I don't think so. I just fixed and pushed. Please check

Reply:
So how should it behave? Can you add a unit test, please? Something like `compose - load contacts through API` but simpler. If we run such a test only for `testVariant === 'ENTERPRISE-MOCK'`, `googleContactsSearchEnabled` will be `true`. You can always find the profile's own email address in contacts, that is `ci.tests.gmail@flowcrypt.test` for `ci.tests.gmail` profile

Reply:
Current behavior is to show `no contact founds` for all cases when there are no contacts found

Reply:
This is current implemented behavior.
@tomholub Not sure who can confirm this. Can you confirm if this is what you are looking for? I'll add ui test (as Roman suggested) after your confirmation

![chrome-capture-2022-7-9](https://user-images.githubusercontent.com/77344191/183671576-74a0bfdd-4ce0-4d31-930e-0435eba1898b.gif)

Reply:
behavior-wise, the screen recording looks ok to me. I haven't read the rest of the conversation

Reply:
@rrrooommmaaa Ready for review

Reply:
The element added by `addBtnToAllowSearchContactsFromGoogle` remains as a leftover after the "No Contacts Found" case, is this intentional?
The look when a contact is found right away:
![image](https://user-images.githubusercontent.com/23335944/183896524-f7c86bd6-721a-4ac9-87d6-3c632eb1a87b.png)
The look when a contact is found after the "No Contacts Found" case:
![image](https://user-images.githubusercontent.com/23335944/183897496-36f71542-c59f-4b5d-93b0-36f3a0eb54bb.png)

Reply:
Thanks for your notice
> The element added by addBtnToAllowSearchContactsFromGoogle remains as a leftover after the "No Contacts Found" case

I don't think so. Just fixed
> is this intentional?